internal/grid/connection.go

```diff
@@ -1406,6 +1412,10 @@ func (c *Connection) handleRequest(ctx context.Context, m message, subID *subHan
 }
 
 func (c *Connection) handlePong(ctx context.Context, m message) {
+	if m.MuxID == 0 && m.Payload == nil {
+		atomic.StoreInt64(&c.LastPong, time.Now().Unix())
```

Comment:
```suggestion
		atomic.StoreInt64(&c.LastPong, time.Now().UnixNano())
```

?

Reply:
The one on `Connection` is secs. The one on `muxClient` is nano. Updated it so I could do the tests without them taking minutes.

Reply:
Friday, April 11, 2262 will be a bad day :D